ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-commands.sh

```diff
@@ -230,6 +244,12 @@ echo "  Enable customer managed key: ${ENABLE_CUSTOMER_MANAGED_KEY}"
 echo "  Enable ec2 metadata http tokens: ${EC2_METADATA_HTTP_TOKENS}"
 echo "  Enable etcd encryption: ${ETCD_ENCRYPTION}"
 echo "  Disable workload monitoring: ${DISABLE_WORKLOAD_MONITORING}"
+if [[ "$ENABLE_PROXY" == "true" ]]; then
+  echo "  Using proxy."
+fi
+if [[ "$ENABLE_PROXY" == "true" ]] && [[ -f "$TRUST_BUNDLE_FILE" ]]; then
+  echo "  Using proxy with additional trust bundle."
+fi
```

Comment:
No need TRUST_BUNDLE_FILE part. Just writing like `echo "  Enable proxy: ${ENABLE_PROXY}` is enough.

Reply:
ok I wasn't sure if we wanted to note in the logs if we are using the trust bundle or not.

Reply:
`TRUST_BUNDLE_FILE` is a supplement point for proxy. In my option, I do not want to separate it from proxy, it means that if it is a proxy, it should include TRUST_BUNDLE_FILE(for TRUST_BUNDLE_FILE on the proxy server is not ready, we now just make a workaround to not use it).

Reply:
fixed